eng/tools/TypeSpecValidation/src/tsv-runner-host.ts

```diff
@@ -11,6 +13,10 @@ export class TsvRunnerHost implements TsvHost {
     return simpleGit(folder);
   }
 
+  readTspConfig(folder: string): Promise<string> {
+    return readFile(join(folder, "tspconfig.yaml"), "utf-8");
```

Comment:
do we need error handling when file not found?

Reply:
I think it's fine to just throw.  Rule `FolderStructure` should already ensure `tspconfig.yaml` exists.